pages/home.page.js

```diff
@@ -3,7 +3,9 @@ import BasePage from '../pages/base.page';
 class HomePage extends BasePage{
 
     get welcomeUser(){return $('.active')}
+    get userMenuButton(){return $('.user-area dropdown')}
+    get logoutMenuOption(){return $('.fa fa-power-off')}
```

Comment:
Review if you can find another selector for this one, as this is a css class mostly oriented to style, having high chances of changing if they decide to represent the button differently. Consider selecting by text, for example.